plugwise/helper.py

```diff
@@ -981,9 +980,11 @@ def _get_appliance_data(self, d_id: str) -> DeviceData:
             if all(item in data for item in ("cooling_ena_switch", "cooling_enabled")):
                 data.pop("cooling_enabled")
 
-        # Don't show cooling_state when no cooling present
-        if not self._cooling_present and "cooling_state" in data:
-            data.pop("cooling_state")
+        # Don't show cooling-related when no cooling present
+        if not self._cooling_present:
+            for item in ("cooling_state", "cooling_ena_switch", "cooling_enabled"):
```

Comment:
Maybe const these in some future update also describing why these, ok for now

Reply:
Since we use the once, a const doesn't make sense. And adds more lines of code :)